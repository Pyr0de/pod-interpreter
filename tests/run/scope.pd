{
	let a = 10;
	print a;
}
let b = 1;
{
	print b;
}

{
	let m = 1;
}
print m;
