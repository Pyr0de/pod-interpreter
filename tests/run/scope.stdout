ExitCode: 66
10
1
